components/elm/src/main/elm_initializeMod.F90

```diff
@@ -71,15 +71,15 @@ subroutine initialize1( )
     use UrbanParamsType           , only: UrbanInput
     use CLMFatesParamInterfaceMod , only: FatesReadPFTs
```

Comment:
Will CLMFatesParamInterfaceMod be renamed in the future?

Reply:
Yes. This will be changed in the future in consultation with FATES team.